skrobot/interfaces/_pybullet.py

```diff
@@ -289,13 +273,15 @@ def wait_interpolation(self, thresh=0.05):
                                                     idx)
                 if abs(velocity) > thresh:
                     wait = True
+                if time.time() - start > timeout:
+                    wait = False
```

Comment:
If this function is timed out, should return `False`.

Reply:
ok, i got it